libkbfs/tlf_journal.go

```diff
@@ -1295,6 +1352,17 @@ func (j *tlfJournal) shutdown() {
 		return
 	}
 
+	// Even if we shut down the journal, its blocks still take up
+	// space, but we don't want to double-count them if we start
+	// up this journal again, so we need to adjust them here.
```

Comment:
I guess if we ever expect to shut down non-empty journals in production (say because the FBO for a TLF gets force-garbage-collected or something), there's the potential to go over the disk limit.  Maybe a TODO here that we should find a way around this, possibly by passing into the constructor whether the journal is being created for the first or Nth time?